src/autoconf/main.py

```
@@ -65,8 +89,31 @@ def exit_handler(signum, frame):
+            logger.info("Kubernetes mode detected")
+            controller = IngressController()
+        else:
+            logger.info("Docker mode detected")
+            controller = DockerController(docker_host)
+
+        if kubernetes:
+            run_on_kubernetes_ha_mode(controller=controller)
+        else:
+            start(controller=controller)
+
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)